packages/vm/Cargo.toml

```diff
@@ -7,6 +7,18 @@ description = "VM bindings to run cosmwams contracts"
 repository = "https://github.com/CosmWasm/cosmwasm/tree/main/packages/vm"
 license = "Apache-2.0"
 
+[package.metadata.release]
+pre-release-hook = [
+    "../../devtools/release_checks.sh"
+]
+pre-release-replacements = [
+    { file="../../CHANGELOG.md", search="## \\[Unreleased\\]", replace="## [{{version}}] - {{date}}", exactly=1 },
+    { file="../../CHANGELOG.md", search="Unreleased", replace="{{version}}" },
+    { file="../../CHANGELOG.md", search="<!-- next-header -->", replace="<!-- next-header -->\n\n## [Unreleased]", exactly=1 },
+    { file="../../CHANGELOG.md", search="\\.\\.\\.HEAD", replace="...{{tag_name}}", exactly=1 },
+    { file="../../CHANGELOG.md", search="<!-- next-url -->", replace="<!-- next-url -->\n[Unreleased]: https://github.com/CosmWasm/cosmwasm/compare/{{tag_name}}...HEAD", exactly=1 },
```

Comment:
The spacing still doesn't work correctly with this. See the newline between `unreleased` and `2.0.2`:
```diff
 <!-- next-url -->
 
-[unreleased]: https://github.com/CosmWasm/cosmwasm/compare/v2.0.1...HEAD
+[unreleased]: https://github.com/CosmWasm/cosmwasm/compare/v2.0.2...HEAD
+
+[2.0.2]: https://github.com/CosmWasm/cosmwasm/compare/v2.0.1...v2.0.2
```
Does this support capture groups? Maybe it's easier to replace the whole `[unreleased]: ...` line with two lines?

Reply:
The newest two commits should fix both of the review comments:

```patch
 ## [Unreleased]
 
+## [2.0.2] - 2024-04-08
+
 ### Added
 
 - cosmwasm-vm: Add `secp256r1_verify` and `secp256r1_recover_pubkey` imports for
@@ -804,7 +806,8 @@
 
 <!-- next-url -->
 
-[unreleased]: https://github.com/CosmWasm/cosmwasm/compare/v2.0.1...HEAD
+[unreleased]: https://github.com/CosmWasm/cosmwasm/compare/v2.0.2...HEAD
+[2.0.2]: https://github.com/CosmWasm/cosmwasm/compare/v2.0.1...v2.0.2
 [2.0.1]: https://github.com/CosmWasm/cosmwasm/compare/v2.0.0...v2.0.1
 [2.0.0]: https://github.com/CosmWasm/cosmwasm/compare/v1.5.0...v2.0.0
 [1.5.0]: https://github.com/CosmWasm/cosmwasm/compare/v1.4.1...v1.5.0
```